CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ Mapbox welcomes participation and contributions from everyone.
 * Preserve cached properties if applied to the layer before during the `Style#getLayer` call. ([1622](https://github.com/mapbox/mapbox-maps-android/pull/1622))
 * Fix a `NullPointerException` in `StandardGestureListener`, where `MotionEvent` should be nullable. ([1645] (https://github.com/mapbox/mapbox-maps-android/pull/1645))
 * Fix incorrect `MapView` dimensions after background orientation change. ([1658](https://github.com/mapbox/mapbox-maps-android/pull/1658))
+* Wrong BitmapWidget position when using WidgetPosition.Horizontal.CENTER. ([1651](https://github.com/mapbox/mapbox-maps-android/pull/1651))
```

Comment:
nit: seems duplicated with the entry below, remove this line?
```suggestion
```

Reply:
oho thanks Peng!